packages/segmentedControl/components/SegmentedControlButton.tsx

```diff
@@ -36,21 +36,21 @@ export interface SegmentedControlButtonProps {
    * Content that appears in a Tooltip when a users hovers the button
    */
   tooltipContent?: React.ReactNode;
+  children?: React.ReactNode;
 }
 
-class SegmentedControlButton extends React.PureComponent<SegmentedControlButtonProps> {
-  placeholderId = nextId("segmentedControlButton-");
+const placeholderId = nextId("segmentedControlButton-");
```

Comment:
I think this needs to be defined inside the component in order for the segmented control to work properly. See suggested change below.